app/assets/stylesheets/alert_editor.scss

```diff
@@ -10,3 +10,7 @@ select.colours option[value="neutral"] {
 select.colours option[value="positive"] {
   background: $bg-positive;
 }
+
```

Comment:
Bootstrap styling not really working in a list item for forms, this works, didn't want to spend more time on it!